client/platform/web-girder/plugins/vuetify.ts

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify/lib';
+import colors from 'vuetify/lib/util/colors';
+import { merge } from 'lodash';
+
+import '@mdi/font/css/materialdesignicons.css';
+
+Vue.use(Vuetify);
+
+function getVuetify(config: unknown = {}) {
```

Comment:
Fine for now (through obscurity) but could be a recipe for disaster in the future if we tell people about it.  We may want to ensure a minimal viable config instead of a merge with priority given to the options.  I could easily set `vuetify: { theme: {} }` in the item and break stuff.  I imagine there are other things I can edit to more subtly break things.  Like we make some direct references to `theme.dark` in places.

Reply:
I was looking at it like user stylesheet support.  In wordpress, and on subreddits, and all over the place, apps allow you to add your own CSS fragments.  You can screw everything up, but it's your problem if you do that.

Unless I'm misunderstanding the risk.